SYSTEMS.md

```diff
@@ -71,6 +71,25 @@ reframe -c examples/sombrero -r --performance-report --system csd3-skylake:compu
 where `<ACCOUNT>` is the project you want to charge.
 You can see the account balance of your projects with the `mybalance` command.
 
+## DIaL2
+
+### Queue options
+
+When submitting jobs to compute nodes, you need to specify the job queue, with the `--account` option to the scheduler.
+To do this, when you run a benchmark you can use the `-J`/`--job-option` flag to `reframe` to specify the account, for example:
+
+```
+reframe -c examples/sombrero -r --performance-report --system dial2:compute-node -J'--accout=<ACCOUNT>'
+```
+
+where `<ACCOUNT>` is the project you want to charge.
+
+This platform possesses the pbs/torque scheduler. In order to obtain and use the correct header option (`#PBS -l nodes=?:ppn=?`)
+the following additional flags are required (tested with sphng, trove, trove-pdsyev mpi/openmp hybrid apps):
+
+* `-S executable='-perhost $NUM_TASKS_PER_NODE <NAME-OF-EXECUTABLE>'`
+* `-S env_vars=I_MPI_JOB_RESPECT_PROCESS_PLACEMENT:off`
```

Comment:
If this is _always_ needed, you can set this environment variable in the partition configuration like https://github.com/ukri-excalibur/excalibur-tests/blob/615beef981c8bc4a1a243c5adedc365f4693929d/benchmarks/reframe_config.py#L48-L55

Reply:
In theory it doesn't need to be set for every job, just those that use intel MPI launcher. Which for our tests (dial tests) is all of them. But in theory it doesn't need to be. However, it should have no effect on openMPI jobs as the env variable should be ignored. So I can add it to the env_vars for dial2.5. Would you prefer that?

Reply:
@giordano also do you want me to default add the env_var for all jobs or leave it as is?

Reply:
If `I_MPI_JOB_RESPECT_PROCESS_PLACEMENT` is needed for all Intel MPI jobs and it's ignored by other MPI implementations, I think it'd still be easier to set it in the partition settings, yes.

Reply:
Thanks @giordano, this particular flag is needed for mpi/opnmp hybrid apps only. 

I am working on a solution and will commit shortly for final review.